Sync stream creation

sevko opened the pull request:
Address #6 .

sevko committed:
Make createLookupStream() pseudo-sync.

lib/master.js
	-Make `createLookupStream()` pseudo-synchronous, returning
	immediately, rather than classically asynchronous as per #6.

sevko committed:
Update createLookupStreamSync() documentation.

lib/master.js
	-Update the documentation to reflect changes in fe084aa.

sevko committed:
Remove original main module.

index.js
	-A high-level module that abstracted away the gory asynchronous
	details of lookup/stream creation `lib/master` only made sense
	before fe084aa.
	-Now, it's a little useless. Remove.

sevko committed:
Update references to old main module.

test/(lookup_results, test, app/server).js
	-Update references to `/index.js`, which was removed in 9475f20.

sevko committed:
Update README API documentation.

README.md
	-Document the recent changes to `stream()`
	(`createLookupStreamSync()`).

sevko commented:
Merging when @dianashk and @missinglink sign off.

sevko committed:
package.json: v1.0.2

dianashk commented on package.json:
```diff
@@ -1,10 +1,10 @@
 {
   "name": "pelias-admin-lookup",
-  "version": "1.0.1",
+  "version": "1.0.2",
```
This isn't a big deal, but don't we usually up the version in a separate commit on master?

sevko replied:
Yeah, this was my bad.

dianashk commented:
+1 (other than package version)

missinglink commented:
This is super awesome, the interface is much cleaner and easier to use, couple comments:

1. it would be best to defend against consumers instantiating 2x streams:

```javascript
adminLookup = require( 'pelias-admin-lookup' )
stream1 = adminLookup.stream();
stream2 = adminLookup.stream();
```

2. you should increment a change to the major version number (ie bump to v2.0.0) due to the breaking API change

sevko committed:
Resolve #8, prevent multiple worker initializations.

lib/master.js
	-Prevent the consumer from initializing the admin-lookup
	multiple times, as per #8 (though it was caused by another issue
	which resulted from multiple `initWorkers()` calls, this
	shouldn't ever happen).

sevko committed:
package.json: v2.0.0
